sephora/game/custom_turfs.dm

```diff
@@ -5,6 +5,46 @@
 	legacy_smooth = TRUE //Override /tg/ iconsmooths
 	smooth = TRUE
 	canSmoothWith = list(/turf/closed/wall/ship,/obj/machinery/door,/obj/structure/window,/turf/closed/wall/r_wall/ship)
+	sheet_type = /obj/item/stack/sheet/duranium
+
+/obj/structure/girder/attackby(obj/item/W, mob/user, params) //Time to add support for our walls..
+	var/obj/item/stack/sheet/S = W
+	if(istype(W, /obj/item/stack/sheet/duranium))
```

Comment:
Unecessary, just do
if(istype(W, /obj/item/stack/sheet))
Because we're checking the wall type that any sheet has, not just duranium

Reply:
Can fix.

Reply:
Done.